backend.py

```diff
@@ -56,9 +56,12 @@ def __init__(self, direction, path, coordinates):
         self.direction = direction
         self.path = path
         self.coordinates = coordinates
+        self.lifecount = 3
+        self.flagcount = 0
+        self.injurycount = 0
 
     def __repr__(self):
-        return "<Robot {} {} {}>".format(self.direction, self.path, self.coordinates)
+        return "<Robot {} {} {} Lifes: {} Flags: {} Damages: {}>".format(self.direction, self.path, self.coordinates, self.lifecount, self.flagcount, self.injurycount)
```

Comment:
Ještě mě napadla taková drobnost. Jako atribut robota používáš `self.injurycount`, ale ve výpisu `Damages:`. Teď jsem se dívala do Terčina kódu k interface a má tam teďka taky `robot_injury`. Vím, že jsem ten výraz 'damage' jsem navrhla já, ale možná bude asi opravdu lepší používat pro zranění robota 'injury' a pro zranění laserem používat 'damage' (ve smyslu, že robot způsobuje zranění svým laserem nebo taky políčko laseru). Nebo co ti přijde lepší?

Jinak je to OK.

Reply:
mě to přišlo lepší, protože zase roboti jako stroje mají spíš poškození.
Pokud chceme rozlišit zranění robotem a laserem, tak by se to dalo rozděli, ale není to kontraproduktivní? Je třeba to rozlišovat? Ke zranění by se možná mohly používat třeba v nějaké fázi stejné funkce, tam by dva názvy nepomohly.

Reply:
To máš asi pravdu. Tak vyber, co ti přijde lepší, sjednoť to a bude 😉